ui/app/components/breadcrumbs/job.hbs

```diff
@@ -49,6 +49,9 @@
           <dd>
             {{this.job.trimmedName}}
           </dd>
+            {{#if this.job.namespace}}
+              <dt>{{this.job.namespace.name}}</dt>
```

Comment:
Semantically, we'd want to avoid using `<dt>` here as it implies something that needs to be defined. I think you'll want to use a less semantic-laden tag (span?) and apply some styles to it in the breadcrumb stylesheet.

Reply:
Ahhhh! But then `span` doesn't create a new line!! Is it OK if I do

```
<br>
<span>Namespace</span>
```